security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,37 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-0"]
+== OpenShift Compliance Operator 1.4.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:7658[RHBA-2023:7658 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Users can now pause scan schedules by setting the `ScanSetting.suspend` attribute to `True`. This allows users to suspend a scan and reactivate it without the need to delete and re-create the `ScanSettingBinding`. This simplifies pausing scans during maintenance periods. link:https://issues.redhat.com/browse/CMP-2123[CMP-2123]
```

Comment:
should we mention here that we are pausing the scan schedule instead of the actual scan?

Reply:
@Vincent056 I see your point, thank you for that feedback. I'll change "suspend a scan" to "suspend a scan schedule"